mgmtfn/dockplugin/netDriver.go

```diff
@@ -190,7 +190,7 @@ func createEndpoint(hostname string) func(http.ResponseWriter, *http.Request) {
 			return
 		}
 
-		log.Infof("CreateEndpointRequest: %+v. Interface: %+v", cereq, cereq.Interface)
+		//log.Infof("CreateEndpointRequest: %+v. Interface: %+v", cereq, cereq.Interface)
```

Comment:
This can be deleted if you are just moving it down to line 217

Reply:
ok; will remove it